packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -1756,4 +1758,105 @@ describe("DeleteAttributeUseCases", () => {
             expect(CoreDate.from(updatedPredecessor.deletionInfo!.deletionDate).isBetween(timeBeforeUpdate, timeAfterUpdate.add(1))).toBe(true);
         });
     });
+
+    describe(DeleteThirdPartyOwnedRelationshipAttributeAndNotifyPeerUseCase.name, () => {
+        let peerSharedRelationshipAttribute: LocalAttributeDTO;
+        let thirdPartyOwnedRelationshipAttribute: LocalAttributeDTO;
+        beforeEach(async () => {
+            peerSharedRelationshipAttribute = await executeFullCreateAndShareRelationshipAttributeFlow(services3, services1, {
+                content: {
+                    value: {
+                        "@type": "ProprietaryString",
+                        value: "My amazing string",
```

Comment:
I have never seen such an amazing string!

Reply:
actually we use descriptive names for strings like this (in this case `aProprietaryString` should be used).

Reply:
:(